tests/test_integrator/test_fiat_adapter.py

```diff
@@ -35,7 +35,8 @@ def test_fiat_adapter_no_measures():
     test_scenario.init_object_model()
     # TODO: Hazard class should check if the hazard simulation has already been run when initialized
     test_scenario.direct_impacts.hazard.has_run = True  # manually change this for now
-    test_scenario.direct_impacts.run_fiat()
+    test_scenario.direct_impacts.hazard.set_sfincs_map_path("single_scenario")
```

Comment:
The only way to check the run is to actually have the sfincs model already finished. If that is the case you do not need to force this parameters, since they should be correctly identified when you initialize the object.

Reply:
This means that both lines *.has_run = True and *set_sfincs_map_path() are not needed